msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -695,5 +679,23 @@ Define the properties your app uses to post a user activity feed.
 ```
 
 ***
->
->
+
+## defaultInstallScope
+
+**Optional** - string
+
+Specifies whether the scope offers an experience in the context of an individual user alone (personal), a team, a groupchat, or meetings. Options are:
+* `personal`
+* `team`
+* `groupchat`
+* `meetings`
+
+## defaultGroupCapability
+
+**Optional** - object
+
+Specifies the default capability for each collaboration scope your app supports. Options are:
```

Comment:
Could you please help to change this?

When a group install scope is selected, this will define the default capability when the user installs the app.